pom.xml

```diff
@@ -1246,14 +1247,32 @@ under the License.
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-release-plugin</artifactId>
-					<version>2.1</version>
+					<version>2.5.3</version>
 					<configuration>
-						<mavenExecutorId>forked-path</mavenExecutorId>
-						<useReleaseProfile>false</useReleaseProfile>
-						<arguments>${arguments} -Psonatype-oss-release</arguments>
+						<releaseProfiles>release</releaseProfiles>
+						<preparationGoals>clean install</preparationGoals>
+						<goals>deploy</goals>
+						<autoVersionSubmodules>true</autoVersionSubmodules>
+						<pushChanges>false</pushChanges>
 					</configuration>
 				</plugin>
 
+				<!-- Rename the artifact produced by the Apache parent's
+				source release. -->
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-assembly-plugin</artifactId>
```

Comment:
Not, sure, actually. I'm setting it manually to make it resemble our current file naming.